### PR TITLE
Add prometheus and envoy images for importing to k3d

### PR DESCRIPTION
**Pull request by limonkufu**
My setup failed on prometheus pull due to network issues(slow speeds, docker hub rate limiting without login) in the venue and this could help with it. I didn't notice any other missing image

### CODE REVIEW

File: create-cluster.sh [hunk 1]
```diff
@@ -47,6 +47,10 @@ for image in \
   envoyproxy/envoy:distroless-v1.29.2 \
   envoyproxy/gateway-dev:72c0cc7 \
   docker.io/envoyproxy/gateway:v1.0.0 \
+  docker.io/prom/prometheus:v2.48.1 \
+  docker.io/envoyproxy/gateway:v1.0.0 \
+  envoyproxy/gateway-dev:72c0cc7 \
+  envoyproxy/envoy:distroless-v1.29.2 \
```
Comment (mikemorris) [inline create-cluster.sh]:
```suggestion
  docker.io/prom/prometheus:v2.48.1 \
```
Looks like the additional three lines are duplicates and we already have those now.

**Review by mikemorris — COMMENTED** [review]
LGTM after applying the suggestion to remove duplicates - @kflynn do you know what would be requiring the prometheus image pull?

**Comment by kflynn** [comment]
That's from LInkerd Viz -- I should've added it eariler!

**Comment by kflynn** [comment]
@limonkufu, many thanks for the PR! Sorry to hear that it didn't go smoothly in the venue. 🙁

**Pull request merged**